packages/template-retail-react-app/app/pages/product-detail/index.test.js

```diff
@@ -488,3 +490,18 @@ describe('product bundles', () => {
         })
     })
 })
+
+test('renders "Add to Cart" button in French', async () => {
```

Comment:
For unit tests in pages,  we want to write more well-rounded tests that spans across components. This test only checks one single component. How about moving this tests to components/product-view/index.jss that renders the addToCart and addToWishList Buttons.

Reply:
There were no tests for localization, so I added one, but I can move this to product-view page and test the translations for addToCart and addToWishList Buttons both

Reply:
Updated the test